src/components/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,134 @@
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+import type { StoryObj, Meta } from '@storybook/react';
+import React from 'react';
+
+import { Accordion } from './Accordion';
+
+export default {
+  title: 'Organisms/Interactive/Accordion',
+  component: Accordion,
+  parameters: {
+    badges: [BADGE.BETA],
+  },
+  args: {
+    children: (
+      <>
+        <Accordion.Button data-testid="accordion-button">
+          <Accordion.Title as="h2">Massa quam egestas massa.</Accordion.Title>
+        </Accordion.Button>
```

Comment:
Putting this here in the example instead of in the code below for clarity

Is there a case to support where the `Accordion.Button` will have something other than an `Accordion.Title` component (that is not a `"h2"`)? We could streamline usage by providing one component in the implementation (e.g., `Accordion.Header` or similar), and hiding the fact that headless has this button/title separation.

In this example, `Accordion.Header` would have the `as` prop if needed, which would get applied to the Disclosure  title sub-component. Thoughts?

Another consideration: I assume all the `Accordion.Title` instances should have the same, so perhaps configuring the component to use in the current `Accordion.Title` could be handled on `Accordion`

Reply:
> In this example, Accordion.Header would have the as prop if needed, which would get applied to the Disclosure title sub-component. Thoughts?
This makes sense but if a consumer wants the style the Title, would they nest the text in another styled element?

>Another consideration: I assume all the Accordion.Title instances should have the same, so perhaps configuring the component to use in the current Accordion.Title could be handled on Accordion
Are you suggesting receiving the title as a prop instead of using this subcomponent structure?

Reply:
> This makes sense but if a consumer wants the style the Title, would they nest the text in another styled element?

I was fuzzy about how much we would want to allow titles to be styled. if the header can only be text, then we could simplify the component interface a bit:

```
<Accordion.Header as="h2" data-testid="accordion-button">Test Text</Accordion.Header>
``` 
=> 
```
<Accordion.Button>
     <Accordion.Title as="h2">Test Text</Accordion.Title>
</Accordion.Button>
```

> Are you suggesting receiving the title as a prop instead of using this subcomponent structure?

I accidentally a word :P "should have the same **header level**"

This might break a pattern, but my thought was to DRY the use of `as="h2"` across an accordion's individual sections by moving **that** prop to `Accordion`. e.g., `<Accordion as="h2">...</Accordion>` which would apply that to the relevant children. Hope that clears it up

Reply:
Ahhhh thanks for the clarification.
I agree with that the `as` can be set in the highest level `Accordion` component and passed down as necessary. 
I think the `Accordion.Title` component does need to accept styling via `className` and allow elements other than text so will keep this sort of structure.

I have a follow up pr coming with the `Accordion` being the wrapper and `Accordion.Row` being the individual rows so I will move the `as` there